### PR TITLE
fix/openapi debug

### PR DESCRIPTION
**Pull request by smartmind-dev**
Update the POST query endpoint description to look neater and easier to read.

### CODE REVIEW

**Comment by nokchalatte** [comment]
Since https://github.com/smartmind-team/thanosql-api/pull/29 is quite an important fix and feature, I think we should wait until it gets merged into `main` before updating this documentation again and merging it to main.

**Review by stevensjcho — APPROVED** [review]
LGTM as https://github.com/smartmind-team/thanosql-api/pull/31 was merged

**Pull request merged**
